fuzzer_build_script/brotli

```diff
@@ -0,0 +1,7 @@
+$CC $CFLAGS -c -std=c99 -I. -I./c/include c/fuzz/decode_fuzzer.c
+
+$CXX $CXXFLAGS ./decode_fuzzer.o  -o $OUT/decode_fuzzer \
+    $LIB_FUZZING_ENGINE ./libbrotlidec.a ./libbrotlicommon.a
```

Comment:
It's actually fine that `make` is used because in most, if not all (am not sure if time sometimes matters), times `make` will only recompile the targets that have change, i.e. the harness

Reply:
+1, let's keep this as more of an "override" option that's used in if the automatic solution fails.